SK.WebApp/src/services/languages.ts

```diff
@@ -0,0 +1,13 @@
+export enum anyLangageCodeKeys {
```

Comment:
Please move this file to `src/language.ts`

Reply:
done